modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/online/v2_0/GeoServerOnlineTest.java

```diff
@@ -33,7 +33,7 @@
 public class GeoServerOnlineTest extends AbstractWfsDataStoreOnlineTest {
 
     public static final String SERVER_URL =
-            "http://localhost:9090/geoserver/wfs?service=WFS&request=GetCapabilities&version=2.0.0";
```

Comment:
Is the test run on a normal maven build (hopefully not)? There are around a few annoying tests that just fail if there is a GeoServer running locally that does not have the layers they expect.

Reply:
Checked, no worries, they are not run at all